src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-daemon-list/service-daemon-list.component.ts

```diff
@@ -185,6 +187,17 @@ export class ServiceDaemonListComponent implements OnInit, OnChanges, AfterViewI
         prop: 'events',
         flexGrow: 5,
         cellTemplate: this.listTpl
+      },
+      {
+        name: $localize`Memory Usage`,
+        prop: 'memory_usage',
+        flexGrow: 1,
+        pipe: this.dimlessBinaryPipe
+      },
+      {
+        name: $localize`CPU %`,
```

Comment:
@avanthakkar Thanks for the additional useful factors added to dashboard, LGTM.
This table also appears in **Cluster-> Services** and service details So there also it appears Is it?
Instead of _CPU %_ can we rename header to user friendly name i.e _CPU usage_ ?

Reply:
> @avanthakkar Thanks for the additional useful factors added to dashboard, LGTM. This table also appears in **Cluster-> Services** and service details So there also it appears Is it? Instead of _CPU %_ can we rename header to user friendly name i.e _CPU usage_ ?

Yeah it's also visible under Cluster-> Services details.  OK, I can do that, it's just I thought it occupies more space in table so I reduced it to %, anyways will make the changes. Thanks